pymongo/server.py

```diff
@@ -140,8 +140,8 @@ def run_operation(
             request_id, data, max_doc_size = self._split_message(message)
 
         if publish:
-            cmd, dbn = operation.as_command(conn)
             assert listeners is not None
+            cmd, dbn = operation.as_command(conn)
```

Comment:
I assume this change was accidental. Can we revert it?

Reply:
Yeah, a vestige of an failed attempt to refactor from earlier.